internal/report/prometheus.go

```diff
@@ -9,22 +9,33 @@ const namespace = "popeye"
 
 // Metrics
 var (
-	score = prometheus.NewGauge(prometheus.GaugeOpts{
+	score = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
@eminugurkenar Wondering if we should call this metric `popeye_cluster_score_total` to keep with recommended naming. Also perhaps change the help here to something like: Popeye's sanitizers overall cluster score.  I don't think we need a separate metric for grade if we add the grade as a tag here, we could probably do it using a single metric that represents the sanitizer score for this cluster as well as the letter grade...

Reply:
grade and score got merged under popeye_cluster_score_total , grade is label now.